src/modules/universe/RadixNodeInfo.ts

```diff
@@ -0,0 +1,49 @@
+import { RadixBytes } from '../atommodel';
+
+export default interface RadixNodeInfo {
+    attempts?: number
+    hash?: string
+    host: {
+        ip: string
+        port: number
+    }
+    id?: string
+    protocols?: Array<string>
+    serializer?: {
+        class_id: number
+    }
+    statistics?: {
+        duration: number
+        traffic_in: number
+        traffic_out: number
+    }
+    system?: {
+        agent: string
+        clock: number
+        key: RadixBytes
+        shards: {
+            low: number
+            high: number
+        }
+        period: 0
+        port: 0
+        serializer: {
+            class_id: number
+        }
+        services: Array<any>
+        version: {
+            agent: number
+            object: number
+            protocol: number
+        }
+    }
+    timestamps?: {
+        active: number
+        banned: number
+        connected: number
+        disconnected: number
+    }
+    version?: {
+        object: number
+    }
+}
```

Comment:
Is there any documentation on this interface somewhere? Confluence or so? I have a few questions on the meaning of some fields, but I realise this is not the correct forum for such questions, and most likely it would be easier to read the documentation myself.

Reply:
There's some info in Damian's unreleased API documentation, link in private. But I just typed 'em as I saw 'em, not sure if it's even up to date. I don't really use any of it asides from the host field.